gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -199,39 +211,62 @@ def add_arguments(self, parser):
               'For instance, [0, None, 1] will become '
               '[0, `null_numeric_value_replacement`, 1].'))
 
+
   def validate(self, parsed_args, client=None):
     # type: (argparse.Namespace, bigquery.BigqueryV2) -> None
     if not parsed_args.output_table and parsed_args.output_avro_path:
       # Writing into BigQuery is not requested; no more BigQuery checks needed.
       return
-
-    project_id, dataset_id, table_id = bigquery_util.parse_table_reference(
-        parsed_args.output_table)
-
-    if not client:
-      credentials = GoogleCredentials.get_application_default().create_scoped(
-          ['https://www.googleapis.com/auth/bigquery'])
-      client = bigquery.BigqueryV2(credentials=credentials)
-
-    bigquery_util.raise_error_if_dataset_not_exists(client, project_id,
-                                                    dataset_id)
-    # Ensuring given output table doesn't already exist to avoid overwriting it.
-    if not parsed_args.append:
-      if parsed_args.update_schema_on_append:
-        raise ValueError('--update_schema_on_append requires --append to be '
-                         'true.')
+    if parsed_args.update_schema_on_append and not parsed_args.append:
+      raise ValueError('--update_schema_on_append requires --append to be '
+                       'true.')
+    if (not parsed_args.sharding_config_path or
+        not parsed_args.sharding_config_path.strip()):
+      raise ValueError(
+          '--sharding_config_path must point to a valid config file.')
+    # Ensuring (not) existence of output tables is aligned with --append value.
```

Comment:
I don't think you are checking that _sample table has already been created here. We need to add that as well.

Reply:
A few line below we add it to `all_output_tables` and we verify it along with all other output tables.